targets/TARGET_STM/pwmout_api.c

```diff
@@ -142,9 +142,23 @@ static void _pwmout_init_direct(pwmout_t *obj, const PinMap *pinmap)
         pin_function(pinmap->pin, pinmap->function);
         pin_mode(pinmap->pin, PullNone);
 
-        obj->period = 0;
-        obj->pulse = 0;
-        obj->prescaler = 0;
+        // Initialize obj with default values (period 550Hz, duty 0%)
```

Comment:
This part is 1-to-1 the code in pwmout_period_us() function, can we simplify and only leave one :) ? Maybe via an helper function

Reply:
Ok, I added an helper function 😉